### DISCUSSION TITLE
Exclude dot directories/files in file selector view?

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1141] [Ideas]
If you have a debian desktop system for multiple years, you end up with many dot files/directories as a matter of course. Most WM file browsers treat these as 'hidden' directories/files by default, and one has to enable 'view hidden files' to make them visible.

Premise: if you download an Amiga file title, be that to ~/Downloads, or you download a .zip archive and extract it to $HOMEDIR, to select/load the file into amiberry, you're greeted with the file selector something akin to this ;

 https://www.youtube.com/watch?v=lOnva4YZEHQ

I cannot fathom a need for amiberry to list these (otherwise 'hidden') directories/files in normal usage (nor any need to view same), and it would help expedite file selectors operations if these were ignored by amiberry's file selector listing.

What do you think?

TIA

**Comment by solskogen** [comment]
I agree on this. Showing hidden files and/or directories could rather be a option to turn on or off.

**Comment by giantclambake** [comment]
It could perhaps be an option in amiberry.conf ...ie; show_hidden_files=no/yes  (defaults to no)

**Reply by midwan** [reply]
That's probably the way to go, I'll open a new issue about this.

**Reply by midwan** [reply]
This is now implemented in both `master` and `preview` branches. I didn't add an option for it, as it seems logical to have this as the default behavior. If one needs access to a hidden directory, the path field can be used to type in such a name there. It's a problem if you don't have a keyboard (only using a controller to navigate), but that's probably an edge case.

**Reply by giantclambake** [reply]
Agreed ~ it seems logical to have this as default behavior, and in practice the edge case you reference would be remote.

**Comment by giantclambake** [comment]
@midwan ~ kudos for picking this up in time for next ~ I just checked this --- navigation of listview is improved, and specifying a hidden directory works as it should ... closing this as resolved =)

**Comment by giantclambake** [comment]
Closing as fixed
